src/contact.ts

```diff
@@ -440,7 +440,7 @@ export class Contact implements Sayable {
   public alias(): string | null
 
   /**
-   * set the alias for contact
+   * Set the alias for contact
```

Comment:
If you use `GET / SET / DELETE the alias for a contact` on line 481, then I suggest here should be `SET`  for unification.
Or, change line 481 to `Get / Set / Delete the alias for a contact`.

It should change as well as `Set` and `Delete`

You decide it.